examples/fluids/problems/blasius.c

```diff
@@ -162,31 +233,40 @@ PetscErrorCode NS_BLASIUS(ProblemData *problem, DM dm, void *ctx) {
   // ------------------------------------------------------
   //               SET UP Blasius
   // ------------------------------------------------------
-  problem->ics.qfunction                       = ICsBlasius;
-  problem->ics.qfunction_loc                   = ICsBlasius_loc;
-
-  CeedScalar Uinf   = 40;          // m/s
-  CeedScalar delta0 = 4.2e-4;      // m
-  CeedScalar theta0 = 288.;        // K
-  CeedScalar P0     = 1.01e5;      // Pa
-  PetscBool  weakT  = PETSC_FALSE; // weak density or temperature
-  PetscReal  mesh_refine_height = 5.9e-4; // m
-  PetscReal  mesh_growth        = 1.08;   // [-]
-  PetscInt   mesh_Ndelta        = 45;     // [-]
-  PetscReal  mesh_top_angle     = 5;      // degrees
+  problem->ics.qfunction        = ICsBlasius;
+  problem->ics.qfunction_loc    = ICsBlasius_loc;
+
+  CeedScalar Uinf               = 40;          // m/s
+  CeedScalar Tinf               = 288.;        // K
+  CeedScalar T_wall             = 400.;        // K
```

Comment:
Should we be consistent here in naming? The last bit is updating `examples/fluids/README.md` and making sure that this code is tested. The three Blasius tests all use `blasius-stgtest.yaml`, which have `stg_use: true`. So I think we need a test with compressible Blasius.

```suggestion
  CeedScalar U_inf              = 40;          // m/s
  CeedScalar T_inf              = 288.;        // K
  CeedScalar T_wall             = 400.;        // K
```

Reply:
At least in the user input, should we spell these out, like `velocity_infinity`, `temperature_wall`? Anyway, I don't love ambiguity in T vs theta for temperature. Full names help a user do things like `git grep -i temperature *.yaml`, though this isn't necessarily what we should optimize for.